frontend/locales/ja/nav.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
I'd suggest we remove these if they aren't real translations. We can use the pseudolocalizations to test and do translation verification testing when we get the real translations back.

Reply:
Let me just check with the globalization team. They provided these as part of their workflow. I'll see if they're flexible.

Reply:
If they're real translations, we can keep them.